### ISSUE TITLE
How to get the sensitivity pruner results?

### ISSUE THREAD

**Issue by OberstWB** [microsoft/nni#5349]
**Describe the issue**:
How to get the final sensitivity pruner results? There is no mask to modelspeedup.

**Environment**:linux
- NNI version:2.10
- Training service (local|remote|pai|aml|etc):
- Client OS:
- Server OS (for remote mode only):
- Python version:3.8
- PyTorch/TensorFlow version:pytorch1.8.1
- Is conda/virtualenv/venv used?:yes
- Is running in Docker?:

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by J-shang** [comment]
Hello @OberstWB , in fact, sensitivity pruner is a really old pruner, and it don't support export masks. A workaround is using the config_list saved under the `checkpoint_dir` with your `base_algo` pruner to get the masks.

**Comment by Lijiaoa** [comment]
@OberstWB Looks like this issue had been resolved. Could you help close it?
